testing-modules/selenium/src/main/java/com/baeldung/selenium/visualregression/DriverManager.java

```diff
@@ -0,0 +1,68 @@
+package com.baeldung.selenium.visualregression;
+
+import org.openqa.selenium.PageLoadStrategy;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.Duration;
+import java.util.HashMap;
+
+import static java.text.MessageFormat.format;
+
+public class DriverManager {
+
+    private WebDriver driver;
+
+    public void startChromeInCloud()  {
+        final String LT_USERNAME = System.getenv("LT_USERNAME");
```

Comment:
We typically don't make variables final in our code examples (even though I agree they should be in production-grade code)

Reply:
updated